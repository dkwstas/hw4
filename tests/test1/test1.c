#include <stdio.h>
#include <string.h>
#include <stdlib.h>
#include <unistd.h>

int main (int argc, char *argv[]) {
    char text[] = "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Hendrerit dolor magna eget est lorem ipsum dolor. Felis bibendum ut tristique et egestas. Turpis cursus in hac habitasse. Nullam non nisi est sit amet facilisis magna. Malesuada fames ac turpis egestas sed. Egestas sed tempus urna et pharetra. Bibendum arcu vitae elementum curabitur vitae nunc sed velit. Elit sed vulputate mi sit amet mauris commodo quis imperdiet. Et magnis dis parturient montes nascetur ridiculus mus mauris vitae. Tempor orci eu lobortis elementum nibh. Dui nunc mattis enim ut. Hac habitasse platea dictumst vestibulum rhoncus est. Justo laoreet sit amet cursus sit amet dictum sit. Praesent tristique magna sit amet purus gravida quis. Adipiscing elit pellentesque habitant morbi tristique senectus et netus et. Tristique magna sit amet purus. Vitae suscipit tellus mauris a diam maecenas sed enim ut. Venenatis a condimentum vitae sapien pellentesque habitant morbi. Dolor magna eget est lorem ipsum dolor sit amet consectetur. Mauris vitae ultricies leo integer malesuada nunc. Fermentum dui faucibus in ornare quam. Adipiscing bibendum est ultricies integer. Consectetur purus ut faucibus pulvinar elementum integer enim neque volutpat. Enim lobortis scelerisque fermentum dui faucibus in ornare quam. Non pulvinar neque laoreet suspendisse. Orci sagittis eu volutpat odio facilisis. Felis donec et odio pellentesque diam volutpat commodo sed egestas. Iaculis nunc sed augue lacus viverra vitae. Massa sapien faucibus et molestie ac feugiat sed. Tincidunt eget nullam non nisi est. In hac habitasse platea dictumst quisque. Ullamcorper sit amet risus nullam eget felis. Pretium fusce id velit ut tortor pretium. Tempor commodo ullamcorper a lacus vestibulum. Eleifend quam adipiscing vitae proin sagittis. Accumsan in nisl nisi scelerisque eu ultrices. Felis imperdiet proin fermentum leo vel. Purus sit amet volutpat consequat mauris nunc. Sit amet volutpat consequat mauris. Nunc pulvinar sapien et ligula ullamcorper malesuada proin libero nunc. Aliquet eget sit amet tellus. Mauris ultrices eros in cursus turpis. Sed ullamcorper morbi tincidunt ornare. Aliquam purus sit amet luctus venenatis. Enim sed faucibus turpis in eu mi. Pharetra diam sit amet nisl. Pellentesque sit amet porttitor eget. Orci dapibus ultrices in iaculis nunc sed augue lacus viverra. Sed tempus urna et pharetra pharetra massa massa. Neque viverra justo nec ultrices dui. Odio eu feugiat pretium nibh ipsum consequat nisl vel pretium. Viverra accumsan in nisl nisi scelerisque eu ultrices. Tortor condimentum lacinia quis vel eros donec ac odio. Vitae semper quis lectus nulla at. Elit duis tristique sollicitudin nibh sit amet commodo nulla facilisi. Sed risus pretium quam vulputate dignissim suspendisse in est. Sed adipiscing diam donec adipiscing tristique risus nec. Vitae congue mauris rhoncus aenean vel elit scelerisque mauris pellentesque. Dolor morbi non arcu risus quis. Tempor nec feugiat nisl pretium fusce. In nisl nisi scelerisque eu. Mi proin sed libero enim sed faucibus turpis in. Vitae semper quis lectus nulla at volutpat diam ut. Nibh nisl condimentum id venenatis a condimentum vitae sapien. Ultrices vitae auctor eu augue ut lectus. Egestas maecenas pharetra convallis posuere morbi leo. Cursus vitae congue mauris rhoncus aenean vel. Arcu odio ut sem nulla. Ipsum faucibus vitae aliquet nec. Adipiscing elit pellentesque habitant morbi tristique senectus. Purus viverra accumsan in nisl nisi scelerisque eu. Odio facilisis mauris sit amet massa vitae tortor condimentum.";

    if (argc != 2) {
        printf("Invalid args.\n");
        return(-1);
    }

    char *token = strtok(text, "i");
    while (token != NULL) {
        write(STDOUT_FILENO, token, strlen(token));
        usleep(100000);
        token = strtok(NULL, "i");
    }

    return(0);
}