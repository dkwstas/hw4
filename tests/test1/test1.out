Lorem psum dolor st amet, consectetur adpscng elt, sed do eusmod tempor ncddunt ut labore et dolore magna alqua. Hendrert dolor magna eget est lorem psum dolor. Fels bbendum ut trstque et egestas. Turps cursus n hac habtasse. Nullam non ns est st amet faclss magna. Malesuada fames ac turps egestas sed. Egestas sed tempus urna et pharetra. Bbendum arcu vtae elementum curabtur vtae nunc sed velt. Elt sed vulputate m st amet maurs commodo qus mperdet. Et magns ds parturent montes nascetur rdculus mus maurs vtae. Tempor orc eu loborts elementum nbh. Du nunc matts enm ut. Hac habtasse platea dctumst vestbulum rhoncus est. Justo laoreet st amet cursus st amet dctum st. Praesent trstque magna st amet purus gravda qus. Adpscng elt pellentesque habtant morb trstque senectus et netus et. Trstque magna st amet purus. Vtae suscpt tellus maurs a dam maecenas sed enm ut. Venenats a condmentum vtae sapen pellentesque habtant morb. Dolor magna eget est lorem psum dolor st amet consectetur. Maurs vtae ultrces leo nteger malesuada nunc. Fermentum du faucbus n ornare quam. Adpscng bbendum est ultrces nteger. Consectetur purus ut faucbus pulvnar elementum nteger enm neque volutpat. Enm loborts scelersque fermentum du faucbus n ornare quam. Non pulvnar neque laoreet suspendsse. Orc sagtts eu volutpat odo faclss. Fels donec et odo pellentesque dam volutpat commodo sed egestas. Iaculs nunc sed augue lacus vverra vtae. Massa sapen faucbus et moleste ac feugat sed. Tncdunt eget nullam non ns est. In hac habtasse platea dctumst qusque. Ullamcorper st amet rsus nullam eget fels. Pretum fusce d velt ut tortor pretum. Tempor commodo ullamcorper a lacus vestbulum. Elefend quam adpscng vtae pron sagtts. Accumsan n nsl ns scelersque eu ultrces. Fels mperdet pron fermentum leo vel. Purus st amet volutpat consequat maurs nunc. St amet volutpat consequat maurs. Nunc pulvnar sapen et lgula ullamcorper malesuada pron lbero nunc. Alquet eget st amet tellus. Maurs ultrces eros n cursus turps. Sed ullamcorper morb tncdunt ornare. Alquam purus st amet luctus venenats. Enm sed faucbus turps n eu m. Pharetra dam st amet nsl. Pellentesque st amet portttor eget. Orc dapbus ultrces n aculs nunc sed augue lacus vverra. Sed tempus urna et pharetra pharetra massa massa. Neque vverra justo nec ultrces d0. Odo eu feugat pretum nbh psum consequat nsl vel pretum. Vverra accumsan n nsl ns scelersque eu ultrces. Tortor condmentum lacna qus vel eros donec ac odo. Vtae semper qus lectus nulla at. Elt dus trstque sollctudn nbh st amet commodo nulla facls. Sed rsus pretum quam vulputate dgnssm suspendsse n est. Sed adpscng dam donec adpscng trstque rsus nec. Vtae congue maurs rhoncus aenean vel elt scelersque maurs pellentesque. Dolor morb non arcu rsus qus. Tempor nec feugat nsl pretum fusce. In nsl ns scelersque eu. M pron sed lbero enm sed faucbus turps n. Vtae semper qus lectus nulla at volutpat dam ut. Nbh nsl condmentum d venenats a condmentum vtae sapen. Ultrces vtae auctor eu augue ut lectus. Egestas maecenas pharetra convalls posuere morb leo. Cursus vtae congue maurs rhoncus aenean vel. Arcu odo ut sem nulla. Ipsum faucbus vtae alquet nec. Adpscng elt pellentesque habtant morb trstque senectus. Purus vverra accumsan n nsl ns scelersque eu. Odo faclss maurs st amet massa vtae tortor condmentum.